modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultGenerator.java

```diff
@@ -127,8 +127,8 @@ private void configureGeneratorProperties() {
         config.processOpts();
         config.preprocessSwagger(swagger);
         // TODO need to obtain version from a file instead of hardcoding it
```

Comment:
@wing328 I can remove this as part of the other language changes I will be making for this PR.